packages/frontapp/src/app/components/SearchByUserName/index.tsx

```diff
@@ -45,7 +46,8 @@ const SearchByUserName = () => {
   const redirectToSearchByKey = (e: React.KeyboardEvent<HTMLImageElement>) => {
     if (e.key === 'Enter') {
       navigate(RoutesTypes.READERS);
-      dispatch(setActiveTab(NavbarItems.SEARCH));
+      dispatch(setActiveTab(NavbarItems.READERS));
+      searchRef.current?.blur();
     }
   };
```

Comment:
Do we need this method? Looks like this component is present only on /readers page in the first place.
Also not sure if blur is needed here. If it is needed, wouldn't it be possible to use `e.target` instead of forward ref?

Reply:
I guess we can remove the method. In case of forwarding ref I wanted to make this for the future changes but I guess I'll do it when its needed

Reply:
But we can not do e.target.blur as currently event listener is on div element